### ISSUE TITLE
Mad Scientist Parrot

### ISSUE THREAD

**Issue by cavpolloosigu** [jmhobbs/cultofthepartyparrot.com#237]
It just came to my attention we don't have a mad scientist parrot!

I'll grant anybody who fulfills this request 100 internet points. :grimacing:

**Comment by jmhobbs** [comment]
Could be a variant of Science Parrot?

![Scientist Parrot](http://cultofthepartyparrot.com/parrots/hd/scienceparrot.gif)

**Comment by cavpolloosigu** [comment]
Yes, but we as it is common knowledge, mad scientists don't need safety googles and have a crazy hairdo. :grin:

**Comment by melinerunen** [comment]
Something like this http://www.wowhead.com/npc=36678/professor-putricide ?
